content/docs/integrating-with-other-libraries.md

```diff
@@ -186,21 +187,21 @@ class Chosen extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/xdgKOz?editors=0010)
+[**Pruébalo en CodePen**](http://codepen.io/gaearon/pen/xdgKOz?editors=0010)
 
-## Integrating with Other View Libraries {#integrating-with-other-view-libraries}
+## Integración con otras librerias de vista
```

Comment:
I suggest keep the capital letters in the titles as the original doc

Reply:
This is common in English, but not quite in Spanish. For consistency, we are using sentence-style capitalization. We could have a further discussion about this when we added to the style guide.